latest.json

```diff
@@ -5,25 +5,29 @@
         "name": "Firefox Beta",
         "release": true,
         "version": "118.0b9",
-        "wmClass": "firefox-beta"
+        "wmClass": "firefox-beta",
+        "channel": "beta"
       },
       "firefox-bin": {
         "name": "Firefox",
         "release": true,
         "version": "117.0.1",
-        "wmClass": "firefox"
+        "wmClass": "firefox",
+        "channel": "release"
       },
       "firefox-esr-bin": {
         "name": "Firefox ESR",
         "release": true,
         "version": "102.15.1esr",
-        "wmClass": "firefox"
+        "wmClass": "firefox",
+        "channel": "release"
```

Comment:
Just checking, these are right? no `channel: "esr"`?

Reply:
This doesn't actually affect anything (yet), we can discuss special-casing ESR in https://github.com/NixOS/nixpkgs/pull/256292